pkgs/tools/misc/as-tree/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "as-tree";
+  version = "0.12.0";
+
+  src = fetchFromGitHub {
+    owner = "jez";
+    repo = pname;
+    rev = version;
+    sha256 = "0c0g32pkyhyvqpgvzlw9244c80npq6s8mxy3may7q4qyd7hi3dz5";
+  };
+
+  cargoSha256 = "0g3x76l251g82jrd194whdsfxzlg45bhr1y6bjjfc79viz9mrr87";
+  cargoPatches = [ ./cargo-lock.patch ];
+  verifyCargoDeps = true;
```

Comment:
```suggestion
```
Someone pointed out to me today that `verifyCargoDeps` [has been removed](https://github.com/NixOS/nixpkgs/issues/79975).

Reply:
Done